providers/snowflake/src/airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -289,8 +315,12 @@ def _get_conn_params(self) -> dict[str, str | None]:
             conn_config["client_id"] = conn.login
             conn_config["client_secret"] = conn.password
             conn_config.pop("login", None)
+            conn_config.pop("user", None)
             conn_config.pop("password", None)
 
+            access_token = self.get_oauth_token(conn_config=conn_config)
+            conn_config["token"] = access_token
```

Comment:
```suggestion
            conn_config["token"] = self.get_oauth_token(conn_config=conn_config)
```
Not a blocker but a small nit.

Reply:
Made this change and pushed.